src/pybind/mgr/dashboard/openapi.yaml

```diff
@@ -7418,550 +7466,6 @@ paths:
       summary: Updates an NFS-Ganesha export
       tags:
       - NFS-Ganesha
-  /api/nvmeof/gateway/info:
-    get:
```

Comment:
This should not be removed

Reply:
that's weird and yeah this shouldn't be removed. It might have got removed because the `ceph-rpm` container in the ceph-dev is missing the `grpcio` libraries which only got added recently. So I think @ivoalmeida needs to update the ceph-dev image (dc pull ceph) and run the command to generate the openapi spec. or if you are generating this without the help of ceph-dev, then you need to install the python3-grpcio rpm, and run the tox command.

Reply:
fixed